test/statsFunctions.js

```diff
@@ -122,6 +122,29 @@ describe('#statsFunctions', () => {
         });
       });
 
+      describe('#timing', () => {
+        it('should send when no dates are specified', done => {
+          server = createServer(serverType, opts => {
+            statsd = createHotShotsClient(opts, clientType);
+            statsd.timing('test', 1592198027348);
+          });
+          server.on('metrics', metrics => {
+            assert.equal(metrics, `test:1592198027348|ms${metricsEnd}`);
+            done();
+          });
+        });
+        it('should send when dates are specified', done => {
+          server = createServer(serverType, opts => {
+            statsd = createHotShotsClient(opts, clientType);
+            statsd.timing('test', new Date(Date.now() - 10));
+          });
+          server.on('metrics', metrics => {
+            assert.equal(metrics, `test:10|ms${metricsEnd}`);
+            done();
+          });
+        });
+      })
```

Comment:
This is missing a semi-colon (which is causing the tests to fail).  Once this is fixed, I can merge this all in, thanks!

Reply:
Damn, did not see that. It did pass on mine when I tested it. Also it'd be better if we put out a prettier config for ppl to use. :) 
I'll update it.